pkg/volume/portworx/portworx_util.go

```diff
@@ -181,13 +186,24 @@ func createDriverClient(hostname string) (*osdclient.Client, error) {
 	}
 }
 
-func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost) (volumeapi.VolumeDriver, error) {
+func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost, localOnly bool) (volumeapi.VolumeDriver, error) {
```

Comment:
I read the docs - and I still don't understand why `CreateDisk` and `DeleteDisk` pass this flag as `false` while others as `true`.